### ISSUE TITLE
Lecture Slides as PDf

### ISSUE THREAD

**Issue by Ovakefali13** [Simulation-Software-Engineering/Lecture-Material#10]
Hey There,

For me personally it would be great to have the slides as PDF, so that I'm able to import them into my OneNote and maybe add some personal annotations to them. I just cloned the Git Repo and was able to create them using pandoc, but since I had some struggle with pandoc when I was starting, maybe it would be a good Idea to have the created PDFs inside the repo? I suggest something for example having a PDFs folder inside the topic folders where these PDFs could be found.

Just an idea/suggestion

**Comment by uekerman** [comment]
Hi!

Importing in OneNote and annotating is certainly a good use case, which I had not in mind. Still, I am a bit hesitant to upload the pdfs to the repository. You can't git diff a PDF :wink: This would give duplicated content.

**Small solution**: explain how to use pandoc instead. Obviously, you need pandoc, but you also need Latex for that.

```bash
pandoc -t beamer -V linkcolor:blue -V fontsize=10pt -s intro_slides.md -o slides.pdf
```

gives decently formatted slides, not very beautiful though.

```bash
pandoc -V geometry:left=1in -V geometry:right=1in -V geometry:bottom=1in -V geometry:top=1in -V linkcolor:blue -V fontsize=10pt -s intro_slides.md -o print.pdf
```

gives a compact DIN A4 version, good if you want to print.

Where did you struggle exactly? (We will teach / work with pandoc in the documentation chapter)

**Proper solution**

Implement a GitHub action that generates the PDFs.

Let's keep this issue open till we have the proper solution. 

Btw, hedgedoc can also convert slides to PDF (I just found out): [example](https://pad.gwdg.de/p/ohrhJiyrx?print-pdf#/)

**Comment by uekerman** [comment]
> This command wouldn't work for the overview.md file

Yes, `overview.md` does not have the `---`, which splits slides.

> Your suggested solution would be much better, and wouldn't that just be a script that runs through all the folders and generates the PDFs with the given command you gave.

Yes, and then providing the PDFs somewhere as an artifact. Here, I am not sure yet what the best solution could be.
